packages/outstatic/src/pages/add-custom-field/index.tsx

```diff
@@ -0,0 +1,557 @@
+                        >
+                          <button
+                            type="button"
+                            onClick={() => {
+                              methods.reset()
+                              setSelectedField(name)
+                              setShowAddModal(true)
+                            }}
+                            className="text-left"
+                          >
+                            <span className="block text-xl cursor-pointer font-bold tracking-tight text-gray-900 capitalize hover:text-blue-500 mb-2">
+                              {field.title}
+                              {/* This span allows for full card click */}
+                              <span className="absolute top-0 bottom-0 left-0 right-16"></span>
+                            </span>
+                            <span className="bg-blue-100 text-blue-800 text-sm font-medium mr-2 px-2.5 py-0.5 rounded">
+                              {field.type}
+                            </span>
+                            {field.required ? (
+                              <span className="bg-red-100 text-red-800 text-sm font-medium mr-2 px-2.5 py-0.5 rounded">
+                                required
+                              </span>
+                            ) : null}
+                          </button>
+                          <button
+                            className="z-10 inline-block text-gray-500 hover:bg-white focus:ring-4 focus:outline-none focus:ring-gray-200 rounded-lg text-sm p-1.5"
+                            type="button"
+                            onClick={() => {
+                              setShowDeleteModal(true)
+                              setSelectedField(name)
+                            }}
+                          >
+                            <span className="sr-only">Delete content</span>
+                            <svg
+                              xmlns="http://www.w3.org/2000/svg"
+                              viewBox="0 0 24 24"
+                              width="24"
+                              height="24"
+                            >
+                              <path fill="none" d="M0 0h24v24H0z" />
+                              <path d="M17 6h5v2h-2v13a1 1 0 0 1-1 1H5a1 1 0 0 1-1-1V8H2V6h5V3a1 1 0 0 1 1-1h8a1 1 0 0 1 1 1v3zm1 2H6v12h12V8zm-9 3h2v6H9v-6zm4 0h2v6h-2v-6zM9 4v2h6V4H9z" />
+                            </svg>
+                          </button>
+                        </div>
+                      )
+                    })}
+                </div>
+              </>
+            )}
+          </>
+        ) : null}
+        {error ? (
+          <div className="mt-8">
+            <Alert type="error">
+              <>
+                <span className="font-medium">Oops!</span> We couldn&apos;t{' '}
```

Comment:
We can surface this error in a more useful way, yes? I don't mind looking in the browser's console, but the average user may not understand what this means.

Reply:
Definitely, we could use some better ux here, I just can't seem to think of a better phrase. Perhaps, "Oops, we couldn't create your custom field. Please, try again later".

Reply:
Yeah, "unable to create your custom field" is probably more actionable. Especially when we get this error message sent back to us in the form of a bug report.